bin/echo.c

```diff
@@ -27,6 +27,48 @@
 #include <errno.h>
 
 #include <s2n.h>
+#include "crypto/s2n_rsa.h"
+#include "stuffer/s2n_stuffer.h"
+#include "utils/s2n_safety.h"
```

Comment:
Might be wrong, but my understanding was that s2nc and s2nd should serve as an example for s2n users. Since public s2n api doesn't expose guards and stuffers, won't it be better to avoid using them here?